settings/R/write.settings.R

```diff
@@ -0,0 +1,34 @@
+##' Takes in a settings object, performs a series of checks, fixes & updates settings and produces pecan.CHECKED.xml
+##'
+##' @title Fix Deprecated Settings
+##' @param settings settings list
+##' @return updated settings list
+##' @author Ryan Kelly
+##' @export write.settings
+
+
+write.settings <- function(settings, outputfile = "pecan.CHECKED.xml"){
+  library(XML)
+  library(PEcAn.DB)
+  library(PEcAn.utils)
```

Comment:
settings2pecan.CHECKED?
Or should we not call the other function update.settings?